tailcfg/tailcfg.go

```diff
@@ -829,6 +831,19 @@ type Hostinfo struct {
 	//       require changes to Hostinfo.Equal.
 }
 
+// VIPService represents a service created on a tailnet from the
+// perspective of a node providing that service. These services
+// have an IP address distinct from the node's IPs.
+// Ports specify which ProtoPorts are made available by this node
+// on the service's IP.
```

Comment:
I'm not following this. How do I find the IP? Can I advertise port 80 for two different services?

Reply:
The IP will come from control later with `PeerCapabilityServicesDestination`, that doesn't exist yet.

Yes, you can advertise port 80 for two different services, and have them to go to different places depending on which service's IP the request comes into.

Reply:
So how does the method above listing VIP services say that "I do 80 on service A, but not on service B"

Or "I'm jellyfin on this 443 but homeassistant on that IP's 443"